components/templates/Function/index.js

```diff
@@ -3,6 +3,10 @@
 var React = require('react');
 
 var TemplateFn = React.createClass({
+  propTypes: {
+    template: React.PropTypes.string,
```

Comment:
No worries, if you do not mind a little red bullet when prototyping then you are all good.

But as soon as you do the PR it will go through the linter yes